scss/_badge.scss

```diff
@@ -1,42 +1,65 @@
-// Base class
 //
-// Requires one of the contextual, color modifier classes for `color` and
-// `background-color`.
+// Base styles
+//
 
 .badge {
   // scss-docs-start badge-css-vars
-  --#{$prefix}badge-padding: #{subtract(var(--#{$prefix}badge-padding-y), 1px) var(--#{$prefix}badge-padding-x) add(var(--#{$prefix}badge-padding-y), 1px)}; // OUDS mod
-  --#{$prefix}badge-padding-x: #{$badge-padding-x};
-  --#{$prefix}badge-padding-y: #{$badge-padding-y};
-  @include rfs($badge-font-size, --#{$prefix}badge-font-size);
-  --#{$prefix}badge-font-weight: #{$badge-font-weight};
-  --#{$prefix}badge-color: #{$badge-color};
-  --#{$prefix}badge-border-radius: #{$badge-border-radius};
+  --#{$prefix}badge-size: #{$ouds-badge-size-medium};
+  --#{$prefix}badge-border-radius: #{$ouds-border-radius-pill};
+  --#{$prefix}badge-count-padding-x: #{$ouds-badge-space-inset};
+  --#{$prefix}badge-count-padding-y: #{$ouds-badge-space-inset};
   // scss-docs-end badge-css-vars
 
   display: inline-block;
-  padding: var(--#{$prefix}badge-padding);
-  @include font-size(var(--#{$prefix}badge-font-size));
-  font-weight: var(--#{$prefix}badge-font-weight);
-  line-height: 1;
-  color: var(--#{$prefix}badge-color);
+  width: var(--#{$prefix}badge-size);
+  min-width: var(--#{$prefix}badge-size);
+  height: var(--#{$prefix}badge-size);
```

Comment:
I just tried without witdh and height, and it seemed better for the Bootstrap compat, by far, I couldn't sport any issue on the OUDS one without, even on zoom.